### DISCUSSION TITLE
box appears for 3 seconds

### DISCUSSION THREAD

**Discussion by wrathofflan** [pubpub-zz/ppInk#302] [Q&A]
I get a box that appears for 3 seconds to tell me the "Length: --.-- Pixels" of whatever line/arrow I've just drawn and the dimensions "(--x--)" for every figure. It's distracting to me and my audience (but mostly just me--nobody else actually paying too close attention) 

I read this page a bunch
https://pubpub-zz.github.io/ppInk/

It repeatedly mentioned (and had a few pictures of) an options window, which seemed like maybe a great place I could start poking around, but don't know how to open that window! (Apparently that's too obvious to have to explain.)

I'm running Windows 11. I looked for ppInk in my Installed App, and it isn't there. I have no idea what I'm doing.

As I already feel like a complete idiot, all are welcome to speak to me like an idiot in your response (if you have time)

**Comment by pubpub-zz** [comment]
a) to open options, you have to look  at the notification area next to the clock
if ppInk is not visible it is folded with other programs , press on the up arrow:
![image](https://github.com/pubpub-zz/ppInk/assets/4083478/bdf2b920-2338-421e-9da7-ebebf7603b45)
and then you will get ppInk icon:
![image](https://github.com/pubpub-zz/ppInk/assets/4083478/7c867084-4500-4a18-a964-a771e9cb917a)
right click on it and you will have access to the option menu item
you will then be able to disable measure while drawing:
![image](https://github.com/pubpub-zz/ppInk/assets/4083478/31a6d82c-0865-4648-a940-a6f1414f7053)

do not hesitate to star the project if you like it😉

**Reply by lulala927** [reply]
I have already unchecked the `Show while drawing` option in measurement tools, but this `Length: xxx Pixel` popup box still appears every time I draw, just like in the screenshot. I'm running ppInk v1.9.100.
<img width="2154" height="1172" alt="image" src="https://github.com/user-attachments/assets/b0f2223c-23c3-49be-b46b-af0316813e8c" />
<img width="328" height="144" alt="660ECE7336A2A9AA896B3194933339CD" src="https://github.com/user-attachments/assets/f9a39b72-330d-42d1-98d3-1861aba83c1d" />
<img width="1658" height="135" alt="image" src="https://github.com/user-attachments/assets/33ee0d00-1fd0-4421-a524-c88995be8a1f" />

**Reply by pubpub-zz** [reply]
Can you confirm it is when you are drawing or when you are flying over previously drawn images ? to completely disable the measure uncheck the box on the border

**Comment by wrathofflan** [comment]
Awesome!! Thanks so much!!!

I actually had to had to deselect a the measurement tool entirely to get the box to disappear  
![measurement tools](https://github.com/pubpub-zz/ppInk/assets/173214628/2c603860-0046-4614-8188-45781557cdff)

...but I never would have got there without you help
